server/src/main/java/io/deephaven/server/table/TableServiceGrpcImpl.java

```diff
@@ -405,8 +403,12 @@ public void getExportedTableCreationResponse(final Ticket request,
                         }
                         authWiring.checkPermissionGetExportedTableCreationResponse(
                                 session.getAuthContext(), request, Collections.singletonList((Table) obj));
-                        responseObserver.onNext(ExportUtil.buildTableCreationResponse(request, (Table) obj));
-                        responseObserver.onCompleted();
```

Comment:
I'm assuming it was on oversight this wasn't in a safelyExecute?

Reply:
It doesn't hurt - the risk is that some work _after_ the call might fail to be performed, but since this is a nonExport() submit, nothing need to be returned and stored for future ticket references.

So we're just mitigating some log messages.

I don't love overly defensive calls like this - but I also really don't love the layers we've had to stack on gRPC to make it usable at all, so I'm good with it.